### DISCUSSION TITLE
Tag major version updates in Grafana docker repository

### DISCUSSION THREAD

**Discussion by Ten0** [grafana/grafana#42796] [Platform]
We are looking to control major version updates of Grafana, but want minor version upgrades to be applied automatically.
For this, versioned packages typically tag images with prefixes of the version numbers, e.g. https://hub.docker.com/_/python?tab=tags&page=1&name=3-

Grafana seems to not do this (https://hub.docker.com/r/grafana/grafana/tags?page=2&name=8 e.g. `8.1.1` exists but `8` doesn't).

Please have the CI also tag the major versions, so people can write their `Dockerfile`s as `FROM grafana/grafana:8` and benefit from auto minor-version upgrades.

**Comment by fl0wx** [comment]
What about: https://github.com/grafana/grafana/discussions/47177#discussioncomment-3157311 ?

**Reply by Ten0** [reply]
Looks like #47177 is pretty much the same issue as this? (although it also suggests to tag minor versions and not only patch, which would indeed probably be the best way to solve this issue)

If that's the case then solving #47177 would solve this too so I guess yes :)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
